operator_ui/src/pages/Jobs/New.tsx

```diff
@@ -0,0 +1,270 @@
+import React from 'react'
+import Radio from '@material-ui/core/Radio'
+import {
+  JobSpecFormats,
+  JobSpecFormat,
+  getJobSpecFormat,
+  stringifyJobSpec,
+  isJson,
+  isToml,
+} from 'utils/jobSpec'
```

Comment:
I wonder if `utils/` is a good place for it? Maybe it should be just `./utils` as AFAICT this file is the only one using it? Or maybe even `./getJobSpecFormat`, `./stringifyJobSpec`, etc

Basically, the idea here is that if this file is the only one using it, then it'd be nice to nuke all the related functions by deleting one folder.

Reply:
Moved them to `./utils`. We can move them back to shared/common place if we need them (i suspect it can happen). Cheers